definitions/features/katello.rb

```diff
@@ -0,0 +1,17 @@
+class Features::Katello < ForemanMaintain::Feature
+  metadata do
+    label :katello
+  end
+
+  def dirs
```

Comment:
The `dirs` is too generic. Can you thing of something more specific? `data_dirs` perhaps?